<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>AXR Browser</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newTabAction"/>
    <addaction name="openAction"/>
    <addaction name="reloadAction"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="previousLayoutStepAction"/>
    <addaction name="nextLayoutStepAction"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="logAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="enableAntialiasingAction"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="preferencesAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="menuDebug"/>
   <addaction name="toolsMenu"/>
   <addaction name="windowMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="navigationToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="backAction"/>
   <addaction name="forwardAction"/>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="reloadAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/browser-ui-refresh.png</normaloff>:/icons/browser-ui-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="previousLayoutStepAction">
   <property name="text">
    <string>Previous Layout Step</string>
   </property>
  </action>
  <action name="nextLayoutStepAction">
   <property name="text">
    <string>Next Layout Step</string>
   </property>
  </action>
  <action name="logAction">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About AXR Browser</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="enableAntialiasingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Antialiasing</string>
   </property>
  </action>
  <action name="preferencesAction">
   <property name="text">
    <string>Options...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="backAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/browser-ui-back.png</normaloff>
     <disabledoff>:/icons/browser-ui-back-disabled.png</disabledoff>:/icons/browser-ui-back.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="forwardAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/browser-ui-forward.png</normaloff>
     <disabledoff>:/icons/browser-ui-forward-disabled.png</disabledoff>:/icons/browser-ui-forward.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="newTabAction">
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>reloadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>closeCurrentTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousLayoutStepAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>previousLayoutStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextLayoutStepAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>nextLayoutStep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>showErrorLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableAntialiasingAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>BrowserWindow</receiver>
   <slot>toggleAntialiasing(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preferencesAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>showPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>BrowserWindow</receiver>
   <slot>updateUIForCurrentTabState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>BrowserWindow</receiver>
   <slot>closeTab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newTabAction</sender>
   <signal>triggered()</signal>
   <receiver>BrowserWindow</receiver>
   <slot>newTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>reloadFile()</slot>
  <slot>previousLayoutStep()</slot>
  <slot>nextLayoutStep()</slot>
  <slot>showErrorLog()</slot>
  <slot>showAbout()</slot>
  <slot>toggleAntialiasing(bool)</slot>
  <slot>showPreferences()</slot>
  <slot>updateUIForCurrentTabState()</slot>
  <slot>closeTab(int)</slot>
  <slot>closeCurrentTab()</slot>
  <slot>newTab()</slot>
 </slots>
</ui>
